### DISCUSSION TITLE
checkboxlist

### DISCUSSION THREAD

**Discussion by photojo** [google/filament#6165] [Q&A]
Hallo!
I'm a beginner in laravel and filament.
I've implemented a filament form with a checkboxlist, described here: [https://filamentphp.com/docs/2.x/forms/fields#checkbox-list](url) My values are saved in the DB as ['apple','banana','kiwi'] when those are checked.
Do you have a more concrete example or tutorial, how get the array FROM the DB and set the boxes to checked?
Tanks a lot.

**Comment by romainguy** [comment]
This is not filamentphp. Wrong project.
